app/models/health/team_performance.rb

```diff
@@ -50,7 +50,7 @@ def team_counts
             id: nil,
             name: name,
             patient_referrals: patient_ids,
-            without_required_qa: patient_ids - with_completed_intake - with_required_qa,
+            without_required_qa: patient_ids - with_required_qa,
```

Comment:
I'm probably missing something, but the note above indicates the patient should be in their outreach period OR have a completed care plan.  I'm not seeing that in this, but maybe that is the equivalent of saying "patient" since those are the only two states they can be in?

Reply:
I pushed that logic down to with_required_qa. I'm confused if you are unclear on that code, or here.